### PR TITLE
Add code-snipped "desktopapp"

### PR DESCRIPTION
**Pull request by Kirides**
Added snippet "desktopapp"

i test a lot of stuff using RoslynPad and sometimes need to create a Desktopapp-environment.

desktopapp-snippet example

i guess this could be changed to only be visible in Windows-Builds (any directions on where to setup platform specific snippets?)

### CODE REVIEW

File: src/RoslynPad.Editor.Shared/SnippetManager.cs [hunk 1]
```diff
@@ -152,6 +152,13 @@ internal sealed class SnippetManager
                 "Console.WriteLine",
                 "Console.WriteLine(${Selection})",
                 "if"
+            ),
+            new CodeSnippet
+            (
+                "desktopapp",
+                "#r Framework-include and await Helpers.RunWpfAsync()",
+                "#r \"framework:Microsoft.WindowsDesktop.App\"\nawait Helpers.RunWpfAsync();\n\n${Selection}",
+                "#r"
```
Comment (aelij) [inline src/RoslynPad.Editor.Shared/SnippetManager.cs]:
```suggestion
                "desktopapp"
```

**Comment by aelij** [comment]
Nice idea :) As for platform, I think you can just add an `if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))`. But you'll have to convert the initializer to a method.

**Comment by Kirides** [comment]
Might've went quite overboard with that :)

**Comment by aelij** [comment]
> Might've went quite overboard with that :)

I would've done the same. Thanks :)

**Pull request merged**